src/lib/types/stores/event-store.ts

```diff
@@ -44,4 +47,5 @@ export interface IEventStore
     queryCount(operations: IQueryOperations[]): Promise<number>;
     setCreatedByUserId(batchSize: number): Promise<number | undefined>;
     getEventCreators(): Promise<Array<{ id: number; name: string }>>;
+    getEventCounts(project: string): Promise<ProjectStatusSchema>;
```

Comment:
it looks weird that getEventsCounts returns ProjectStatusSchema. Maybe it should getEventCountDistribution or getDailyEventCounts and match the type name to event counts domain naming?